### PR TITLE
DOCS: Document new option for specifying the directory to which gppkg writes temporary files

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**

### CODE REVIEW

File: gpdb-doc/markdown/utility_guide/ref/gppkg.html.md [hunk 1]
```diff
@@ -77,6 +77,9 @@ sync [<command_options>]
 -h | --help
 :   Display the online help.
 
+-t | --temp
```
Comment (Sasasu) [inline gpdb-doc/markdown/utility_guide/ref/gppkg.html.md:80]: sorry, I did not update the document. the option should be `--tempdir` and the shortcut (`-t`) does not exist.

**Review by Sasasu — APPROVED** [review]

**Review by Sasasu — CHANGES_REQUESTED** [review]
please hold on. the env should be `TMPDIR` not `TEMPDIR`. I will ask others. and the API maybe get modified.

File: gpdb-doc/markdown/utility_guide/ref/gppkg.html.md (hunk 1 above)
Comment (Sasasu) [inline gpdb-doc/markdown/utility_guide/ref/gppkg.html.md:80]:
```suggestion
--tmpdir
```

**Review by Sasasu — APPROVED** [review]

File: gpdb-doc/markdown/utility_guide/ref/gppkg.html.md (hunk 1 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/utility_guide/ref/gppkg.html.md:80]: @radarwave 

I'm confused. Why are we giving the user two ways to configure a temporary directory. Are we saying they can do it either through an environment variable or by passing a flag? Does one override the other? I need more information.

**Review by dyozie — APPROVED** [review]

**Pull request merged**